### PR TITLE
bugfix tests/redfish10/test_redfish10_api_eventservice.py

### PR DESCRIPTION
**Pull request by gpaulos**
Revised response codes in this FIT test to correct values.

@hohene @johren @jimturnquist @diontje @larry-dean

### CODE REVIEW

File: test/tests/redfish10/test_redfish10_api_eventservice.py [hunk 1]
```diff
@@ -52,12 +52,12 @@ def test_redfish_v1_eventservice_subscriptions_post_get_delete(self):
         api_data = fit_common.rackhdapi('/redfish/v1/EventService/Subscriptions/' + api_data['json']['Id'])
         self.assertEqual(api_data['status'], 200, "Was expecting code 200. Got " + str(api_data['status']))
         api_data = fit_common.rackhdapi('/redfish/v1/EventService/Subscriptions/' + api_data['json']['Id'], action="delete")
-        self.assertEqual(api_data['status'], 200, "Was expecting code 200. Got " + str(api_data['status']))
+        self.assertEqual(api_data['status'], 204, "Was expecting code 200. Got " + str(api_data['status']))
```
Comment (hohene) [inline test/tests/redfish10/test_redfish10_api_eventservice.py]: Change the expecting code '200' to 204.

File: test/tests/redfish10/test_redfish10_api_eventservice.py [hunk 2]
```diff
@@ -52,12 +52,12 @@ def test_redfish_v1_eventservice_subscriptions_post_get_delete(self):
...
 
     def test_redfish_v1_eventservice_actions_submittestevent_post(self):
         payload = {}
         api_data = fit_common.rackhdapi('/redfish/v1/EventService/Actions/EventService.SubmitTestEvent', action="post", payload=payload)
-        self.assertEqual(api_data['status'], 200, "Was expecting code 200. Got " + str(api_data['status']))
+        self.assertEqual(api_data['status'], 202, "Was expecting code 200. Got " + str(api_data['status']))
```
Comment (hohene) [inline test/tests/redfish10/test_redfish10_api_eventservice.py]: same here, change 200 to 202 in the comment.

**Comment by hohene** [comment]
+1 after minor comment fixes.

**Comment by gpaulos** [comment]
@keedya

File: test/tests/redfish10/test_redfish10_api_eventservice.py (hunk 2 above)
Comment (gpaulos) [inline test/tests/redfish10/test_redfish10_api_eventservice.py]: OK thanks for catching that. Fixed now.

**Comment by johren** [comment]
test this please

**Pull request merged**